server/api/controllers/fundraise/get-fundraise.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  friendlyName: 'Get Fundraise Campaign object for user purposes',
+
+  description: 'Get Fundraise Campaign object for user purposes',
+
+  inputs: {
+    campaignName: {
+      type: 'string',
+      required: true,
+    },
+  },
+
+  exits: {},
+
+  fn: async function ({ campaignName }, exits) {
+    try {
+      if (campaignName) {
+        const data = await Fundraise.find({
+          campaignName,
+          isDeleted: false,
+        });
+
+        return exits.success(data);
+      }
+      return exits.invalid('Missing required field: campaignName');
```

Comment:
Minor, can leave it if no time - but this is not necessary as you've made the parameter required above

Reply:
Done